src/components/logos/near-logo.tsx

```diff
@@ -1,23 +1,21 @@
 import React, { SVGProps } from 'react';
+import Spacer from '../helpers/spacer';
 
-function NearLogo(props: Partial<SVGProps<SVGSVGElement>>) {
+function NearLogo({ jumbotron }: { jumbotron: boolean }) {
+  const logoText = jumbotron ? <h1>Near</h1> : <h2>Near</h2>;
```

Comment:
Do we want to keep this capitalised?

```suggestion
  const logoText = jumbotron ? <h1>NEAR</h1> : <h2>NEAR</h2>;
```